### DISCUSSION TITLE
Is it possible to build a single article in order to get a list of invalid cross references quickly

### DISCUSSION THREAD

**Discussion by Stoffelche** [dotnet/docfx#8338] [Q&A]
In my situation the build-step alone takes about 13 minutes having about 5800 api files.

BTW is this to be expected? I mean its really slow. (The number is from a build with the new 2.6 version needing 773 seconds, the previous version needed 886 seconds so this is an improvement of may be 15%).

So while authoring an article with xrefs inside, I would really like to have some way to obtain my invalid references faster than this.

**Comment by paulushub** [comment]
Maybe a custom logger.
